README.md

```diff
@@ -81,19 +81,21 @@ The edition of MSSQL to install. Use one of the following values:
   For more information, see
   <https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver15>.
 
-Default: `None`
+Default: `null`
 
 ### `mssql_tcp_port`
 
 The port that MSSQL listens on.
+By default, MSSQL listens on the `1443` TCP port.
 
-Default: `1443`
+Default: `null`
 
 ### `mssql_ip_address`
 
 The IP address that MSSQL listens on.
+By default, MSSQL listens on the `0.0.0.0` IP address.
 
-Default: `0.0.0.0`
+Default: `null`
```

Comment:
Shouldn't the default read `0.0.0.0` then, if it is the value used as the default?

Reply:
I was thinking that if mssql itself provides the default value, we should use `null` here, rather than duplicating the mssql default value in our code.  So that means there will be a difference between what we use in README.md and what we have in `defaults/main.yml` when documenting the default values for some parameters.

Reply:
When no value is provided the IP would be set to 0.0.0.0. Changing this is complicated because i need to know whether the user has defined this variable or not. The variable itself has no default value, it's MSSQL that will use the 0.0.0.0 IP address by default. Setting this to 0.0.0.0 might add confusion because it's then not clear if it overwrites the set IP address setting or not. I would leave it as is.

Reply:
The readme on the line above says ` By default, MSSQL listens on the `0.0.0.0` IP address.`. The default value for the role itself though. is null

Reply:
> Shouldn't the default read 0.0.0.0 then, if it is the value used as the default?

Hmm, probably not, because then if the role is applied against a configured server without this parameter set, it would change the parameter to the default, which is probably not desirable. Still, it looks a bit confusing to write "By default <something> and just after "Default: <something else>".

Reply:
Right - so do we document the actual default value in the README.md which is different than the default value in `defaults/main.yml`?

Reply:
I can keep the `default: null` line and explicitly say that MSSQL sets 0.0.0.0 by default. And the role by default does not change the IP address unless it is specified with this variable. The same with the TCP port setting.

Reply:
So say something like this?  "The default is `null` which means the MSSQL default value will be used, which is currently `0.0.0.0`".  And replace `0.0.0.0` with the port when documenting the port value.

Reply:
I explicitly described it for TCP and IP address vars

Reply:
I think it is clear now, thanks.